File: MasterLogicaProgramacion/EjerciciosPython/Ej4.py
def invertirFraseAMano(frase):
    fraseInvertida = ""

    for i in range(len(frase) - 1, -1, -1):
        fraseInvertida += frase[i]

    return fraseInvertida


print(invertirFraseAMano("HOLE"))

File: MasterLogicaProgramacion/EjerciciosJavascript/Ej4.js
function invertirFraseAMano(frase){
    let fraseInvertida = "";

    for (let i = frase.length - 1; i >= 0; i--) {
        fraseInvertida += frase[i];
    }

    return fraseInvertida;
}

console.log(invertirFraseAMano("HOLE"));